src/Customers.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+/**
+ * Customers.php
+ * php version 7.4.3
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Glenda <glenda@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+
+namespace Xendit;
+
+use Xendit\Exceptions\InvalidArgumentException;
+
+/**
+ * Class Customers
+ *
+ * @category Class
+ * @package  Xendit
+ * @author   Glenda <glenda@xendit.co>
+ * @license  https://opensource.org/licenses/MIT MIT License
+ * @link     https://api.xendit.co
+ */
+class Customers
+{
+    use ApiOperations\Request;
+
+    /**
+     * Instantiate base URL
+     *
+     * @return string
+     */
+    public static function classUrl()
+    {
+        return "/customers";
+    }
+
+    /**
+     * Send a create customer request
+     *
+     * @param array $params user's parameters
+     *
+     * @return array please check for responses parameters here
+     * https://developers.xendit.co/api-reference/?bash#create-customer
+     * @throws Exceptions\ApiException
+     */
+    public static function createCustomer($params = [])
+    {
+        $requiredParams = ['reference_id', 'given_names'];
```

Comment:
what about the situation with mobile_number and email parameter where the merchants need to provide one of them? can we validate it here in the SDK layer so the experience will be consistent with other required params.

Reply:
Aah I see, okay I'll try

Reply:
@nazieb I've pushed my changes regarding this matter. Is it correct? Or is it a better way to handle this?